src/rect.rs

```diff
@@ -420,6 +420,47 @@ impl Rect {
     pub fn to_rounded_rect(self, radius: f64) -> RoundedRect {
         RoundedRect::from_rect(self, radius)
     }
+
+    /// Returns the largest possible `Rect` that is fully contained in this rect with the fixed
+    /// aspect ratio given by `aspect_ratio`.
```

Comment:
```suggestion
```suggestion
    /// Returns the largest possible `Rect` that is fully contained in `self` with the given `aspect_ratio`.
    ///
    /// The aspect ratio is specified fractionally, as `height / width`.
```
    /// aspect ratio given by `aspect_ratio`.
```